app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java

```diff
@@ -118,9 +119,10 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
 
     private void validateEmail() {
 
-        if (mEditTextEmail.getText().toString().trim().length() > 0) {
+        if (mEditTextEmail.getText().toString().trim().length() > 0 &&
```

Comment:
use `mEditTextEmail`'s  length to show error and filter the error message according to the email regex